tests/test_utils.py

```diff
@@ -869,25 +869,37 @@ def test_get_service_instance_list_ignores_underscore():
         assert sorted(expected) == sorted(actual)
 
 
-def test_get_tron_instance_list_from_yaml_finds_actions_properly():
+@mock.patch('paasta_tools.utils.service_configuration_lib.read_extra_service_information', autospec=True)
+@mock.patch('paasta_tools.utils.service_configuration_lib._read_yaml_file', autospec=True)
+def test_load_tron_yaml_empty(mock_read_file, mock_read_service_info):
+    mock_read_file.return_value = {}
+    mock_read_service_info.return_value = {}
+    soa_dir = '/other/services'
+
+    with pytest.raises(utils.NoConfigurationForServiceError):
+        utils.load_tron_yaml('foo', 'dev', soa_dir=soa_dir)
+
+    assert mock_read_service_info.call_count == 1
+    mock_read_service_info.assert_has_calls([mock.call('foo', 'tron-dev', soa_dir)])
+
+
+def test_get_tron_instance_list_from_yaml_with_dicts():
```

Comment:
Are you going to follow up with more changes so that `actions` is always a dict? Otherwise, I'd prefer to keep 2 test cases, one where actions is a list and one where it's not.

Reply:
I'll add the lists test